sdk/profiles/src/main/java/io/opentelemetry/sdk/profiles/data/AggregationTemporality.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.profiles.data;
+
+/**
+ * Specifies the method of aggregating metric values.
+ *
+ * @see "profiles.proto::AggregationTemporality"
+ */
+public enum AggregationTemporality {
```

Comment:
hmm. not sure how I feel about making a copy of this. Perhaps we need to move it to a common module, rather than having a separate copy for profiles.

Reply:
code comment added.